### PR TITLE
added Keys enpoints; updated Authorisation to work correctly with JWT

### PR DESCRIPTION
**Pull request by krylataya**

### CODE REVIEW

File: API/swagger.json [hunk 1]
```diff
@@ -7,21 +7,16 @@
   },
   "host": "dcp.testpoint.io",
   "schemes": [
-    "https"
+    "https",
+    "http"
```
Comment (monkeypants) [inline API/swagger.json]: no, let's forbid HTTP.

File: API/swagger.json [hunk 2]
```diff
@@ -336,6 +339,172 @@
...
+        "tags": [
+          "keys"
+        ],
+        "summary": "Get keys information",
+        "description": "Returns information about keys for particular business",
+        "operationId": "getKeysInfo",
+        "parameters": [
+          {
+            "$ref": "#/parameters/participantIdUrlParameter"
+          }
```
Comment (monkeypants) [inline API/swagger.json]: we also have `?{filter}` parameters, do they belong here too?

Comment (krylataya) [inline API/swagger.json]: not implemented yet, I'll update when it'll be ready

File: API/swagger.json (hunk 1 above)
Comment (krylataya) [inline API/swagger.json]: made new commit - http deleted

**Pull request merged**